Wabbajack.Lib/Downloaders/NexusDownloader.cs

```diff
@@ -75,12 +77,13 @@ public override bool IsWhitelisted(ServerWhitelist whitelist)
                 return true;
             }
 
-            public override void Download(Archive a, string destination)
+            public override async Task Download(Archive a, string destination)
```

Comment:
Code refactors like this are where one potential danger lies.  A previously `void` function that now is `async Task` is the proper swap, but the compiler will not reliably complain if it hadn't been changed and was left void.  So the potential exists that some other function out there needs to be `async Task` but is still void.